test/base/SynchronizedTest.ooc

```diff
@@ -1,39 +1,39 @@
 use ooc-base
 import threading/Thread, os/Time
 
-TestClass: class {
-	refCounter: ReferenceCounter
-	init: func {
-		refCounter = ReferenceCounter new(this)
+version(debugTests) {
+	TestClass: class {
+		refCounter: ReferenceCounter
+		init: func {
+			refCounter = ReferenceCounter new(this)
+		}
+		dispose: func {
```

Comment:
This whole test is weird. I don't know why the diff is so crazy in this file. Do `?w=1` and you'll see it's not this PR.